### DISCUSSION TITLE
windows下编译.exe如何搭建编译环境

### DISCUSSION THREAD

**Discussion by pengyanxin** [hyOzd/serialplot#49] [General]
请问我想编译windowss使用的.exe要如何搭建编译环境

**Comment by hyOzd** [comment]
I haven't built Serialplot under windows for a long time. I cannot help you much. 

You can use the QtCreator to build the serialplot. Only external dependency is the Qwt library. If you use QtCreator you have to somehow build/provide the qwt library yourself.
